src/display/editor/stamp.js

```diff
@@ -133,8 +133,54 @@ class StampEditor extends AnnotationEditor {
     ) {
       this._editToolbar.hide();
       this._uiManager.editAltText(this, /* firstTime = */ true);
-    } else {
-      this.div.focus();
+      return;
+    }
+
+    if (
+      !this._uiManager.useNewAltTextWhenAddingImage &&
+      this._uiManager.useNewAltTextFlow &&
+      this.#bitmap
+    ) {
+      // The alt-text dialog isn't opened but we still want to guess the alt
+      // text.
+      this.#mlGuessAltText();
+    }
+
+    this.div.focus();
+  }
+
+  async #mlGuessAltText() {
```

Comment:
*Possible a stupid question, since I've not looked in detail at the code:*

This method seems to duplicate *some* of the code/logic also found in https://github.com/mozilla/pdf.js/blob/5e052af3f6389a0e7014f81a7cb75ed339577647/web/new_alt_text_manager.js#L212, would it be possible to use this method there as well?